cloudify_cli/cli/cfy.py

```diff
@@ -256,40 +246,77 @@ def parse_and_validate_label_to_delete(ctx, param, value):
         raise CloudifyValidationError(
             'ERROR: The `{0}` argument is empty'.format(param.name))
 
-    if ',' in value or value.count(':') > 1:
+    labels_list = get_formatted_labels_list(value, allow_only_key=True)
+    if len(labels_list) > 1:
         raise CloudifyValidationError(
             'LABEL can be either <key>:<value> or <key>')
+    [(label_key, label_value)] = labels_list[0].items()
+    if label_value:
+        return labels_list[0]
+    return label_key
 
-    label = value.split(':')
-    if len(label) == 1:
-        validate_param_value('The provided key', label[0])
-        return label[0]
-    else:
-        label_key, label_value = label
-        validate_param_value('The key of the provided label', label_key)
-        validate_param_value('The value of the provided label', label_value)
-        return {label_key: label_value}
 
+def get_formatted_labels_list(raw_labels_list, allow_only_key=False):
+    labels_list = []
+    if '\x00' in raw_labels_list:
```

Comment:
I'd bar any ASCII lower than `\x20` really, they can easily mess things up (imagine how the label's display might look with them). But that of course depends of the customer's needs

Reply:
Same for `\x7F` (DELETE)

Reply:
Labels won't contain any control characters.